public/locales/en/layout/modals/about.json

```diff
@@ -1,7 +1,7 @@
 {
-	"description": "Homarr is a <strong>simple</strong> and <strong>modern</strong> homepage for your server that helps you access all of your apps in one place. It integrates with the apps you use to display useful information or control them. It's easy to install and supports many different devices.",
+	"description": "Homarr is a <strong>sleek</strong>, <strong>modern</strong> dashboard that puts all of your apps and services at your fingertips. With Homarr, you can access and control everything in one convenient location. Homarr seamlessly integrates with the apps you've added, providing you with valuable information and giving you complete control. Installation is a breeze, and Homarr supports a wide range of deployment methods.",
```

Comment:
I don't like this

Reply:
This has already been approved and added to the docs as the new description. It's been migrated to this everywhere.

Reply:
Then change it everywhere aha the punctuation is weird and I don't like giving good adjective before the name. Homarr is just a dashboard, being sleek and modern is normal

Reply:
Please see Discord for the options I provided.